libs/ui/src/setup_card_reader_page.test.tsx

```diff
@@ -1,27 +1,29 @@
 import React from 'react';
-import { render } from '@testing-library/react';
+import { render, screen } from '@testing-library/react';
 
 import { SetupCardReaderPage } from './setup_card_reader_page';
 
 describe('renders SetupCardReaderPage', () => {
   test('with no useEffect trigger as expected', () => {
     const { container } = render(<SetupCardReaderPage />);
+    screen.getByText('Card Reader Not Detected');
+    screen.getByText('Please ask a poll worker to connect card reader.');
+    expect(container.firstChild).toMatchSnapshot();
```

Comment:
This looks like a duplicate of the next line. Both snapshots can be removed.

Reply:
Removing duplicate, but this is a case where I don't really see the harm in having a snapshot. Keeping one for now. If I were to go and edit this component, I think I would find it useful to be able to see styles changing.